docs/finishedLesson/finishedLesson-schema.yaml

```diff
@@ -0,0 +1,55 @@
+definitions:
+  finishedLessons:
+    type: array
+    items:
+      type: object
+      properties:
+        _id:
+          type: string
+        lesson:
+          type: string
+          ref: '#/definitions/lesson'
+        status:
+          type: string
+          enum:
+            - inProgress
+            - onHold
+            - completed
```

Comment:
As I see on our UML diagram, we have progress, finished, assigned status.
Fix it.

Reply:
I guess we should think about the naming of these statuses

Reply:
I renamed all satus as in UML diagram except progress. Anatoliy said  tha it was better to change progress to inProgress and I agree with that